src/components/LabelButton/LabelButton.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+
+import { MdOutlineLabel } from "../../utils/icons/icons";
+import "./label-button.css";
+
+function LabelButton({ noteData, setNoteData }) {
+  const [newLabel, setNewLabel] = useState("");
+  const [showDropdown, setShowDropdown] = useState(false);
+  console.log(noteData);
```

Comment:
Remove this log

Reply:
this log is to display the noteData state, will remove it before the merge